registry/repository.go

```diff
@@ -30,6 +30,9 @@ type SignatureManifest struct {
 	Annotations map[string]string
 }
 
+// var for unit testing
+var fetchAll = content.FetchAll
```

Comment:
Can we use a mock service instead? E.g. https://github.com/googlemaps/google-maps-services-go/blob/391036138cfed529e87aaad49922e96da4a393da/directions_test.go#L41

Reply:
updated.